ceph/ceph_admin/bootstrap.py

```diff
@@ -43,6 +43,15 @@ def construct_registry(cls, registry: str, json_file: bool = False):
     # Todo: Retrieve credentials based on registry name
     _config = get_cephci_config()
     cdn_cred = _config.get("registry_credentials", _config["cdn_credentials"])
+    registry_list = (
```

Comment:
Can we keep this in utils or helper module available for whole CI.

Reply:
Hi Sunil,
If we make changes to .cephci.yaml. 
it will be just one line of code. we can remove from 46-50. so i haven't added special function.